CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 20220421
+
```

Comment:
there's been 3 changes to the ami recipes since the last release we should also mention here: https://github.com/aws/amazon-ecs-ami/commits/main

Reply:
```
* bug fix: nvidia package install change to fix issue when running a yum update [#47](https://github.com/aws/amazon-ecs-ami/pull/47)
* bug fix: fixed AL_NAME for AL2022 scripts [#46](https://github.com/aws/amazon-ecs-ami/pull/46)
* enhancement: update the SSM Agent version to 3.1.1260.0 [#45](https://github.com/aws/amazon-ecs-ami/pull/45)
```